apps/script/HoC2023ScriptFiles/HoC2023AiHelperFunctions.py

```diff
@@ -1,16 +1,33 @@
 import json
-from openai.embeddings_utils import (
-    get_embedding,
-    distances_from_embeddings,
-    tsne_components_from_embeddings,
-    chart_from_components,
-    indices_of_nearest_neighbors_from_distances,
-)
 from constants import *
 import pandas as pd
-import pickle 
-import openai
-openai.api_key = '' # Replace with your OpenAI API key
+import pickle
+from scipy.spatial.distance import cosine
+
+# AWS Services Python Libraries
+import boto3
+import botocore
+
+import sys
+import os
+
+module_path = ".."
+sys.path.append(os.path.abspath(module_path))
+
+# KNOWN ISSUE: Their version of utils was a custom library file unique to their bedrock tutorial located here: https://github.com/aws-samples/amazon-bedrock-workshop/blob/109ed616fd14c9eb26eda9bef96eb78c490d5ef6/utils/bedrock.py
+# While it WAS working before, it doesn't work now and the code contains within the utils folder will need to be migrated out to get this working locally
```

Comment:
Thanks for the update, Tyrone. Could you giving a brief description of how you've been running this script on the AWS platform when it was working?

Reply:
Thanks for meeting with me offline and sharing how you've been running the updated script. Looks good!